src/nvim/option_defs.h

```diff
@@ -319,6 +319,7 @@ static char *(p_bo_values[]) = {"all", "backspace", "cursor", "complete",
 #define BO_WILD   0x40000
 
 EXTERN char_u   *p_bsk;         /* 'backupskip' */
+EXTERN int p_bracketedpaste;    /* 'bracketedpaste' */
```

Comment:
You must not use `/*`. Change this comment and all of the adjacent ones to `//` style or you will not pass linter.